### DISCUSSION TITLE
The link to the quickstart guide from the website seems to have broken

### DISCUSSION THREAD

**Discussion by Lodewic** [viewflow/viewflow#383] [General]
Hi, 

I would like to give Viewflow a try because of the demo showing nice use of Material UI components. But following the link to the quickstart guide, https://docs-next.viewflow.io/overview/quick_start.html, yields no result.

**Comment by kmmbvnr** [comment]
I apologize for the broken link in the quickstart guide. The correct link is actually https://docs.viewflow.io/overview/quick_start.html

I'm actively working on a stable release of Viewflow 2.x, and moving resources as soon as they became ready

**Comment by kmmbvnr** [comment]
Cookbook and site links been updated.
